### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by marcin-krolik**
Updates documentation and example

Summary of changes:
- removed `$SNAP_PATH`'
- updated example
- capitalized Snap
- removed `passthru` from example

@intelsdi-x/plugin-maintainers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,31 +78,32 @@ $ http://DOCKERHOST:9200/_cluster/health?pretty
 }
 ```
 #### To build the plugin binary:
-Get the source by running a `go get` to fetch the code:
+
+Fork https://github.com/intelsdi-x/snap-plugin-collector-elasticsearch
```
Comment (katarzyna-z) [inline README.md]: Could you add new line after this sentence it will look better? Now this line is is joined with the next line.

Comment (marcin-krolik) [inline README.md]: @katarzyna-z done

**Comment by katarzyna-z** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -146,29 +129,29 @@ In the node level, this plugin collects metrics listed the next catalog.
 |script| Computing the grades stats based on a script|
 
 ### Examples
-Example running snap-plugin-collector-elasticsearch, passthru processor, and writing data to a file.
+Example running snap-plugin-collector-elasticsearch and writing data to a file.
 
 ![Dockerized example](http://media.giphy.com/media/3osxY87TeMy7jGrbDW/giphy.gif)
```
Comment (kindermoumoute) [inline README.md]: About the gif:
- It's still using $SNAP_PATH.
- It's using `snapd/snapctl`.

Do we need this gif?
If yes, can we update it to the new example?

**Pull request merged**